packages/paste-website/src/pages/components/anchor/index.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Anchor
+description: The anchor can be used to hyperlink to another URL. It accepts both internal and external URLs.
+---
+
+<content>
```

Comment:
I've merged some stuff so now you should be able to use the `ComponentHeader` like on the button page

Reply:
👍